src/plugins/textEmoji/index.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import "./styles.css";
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+export default definePlugin({
+    name: "TextEmoji",
+    description: "Prevents Discord from replacing emoji with images, leaving it up to the browser to render them",
+    authors: [Devs.Grzesiek11],
+    patches: [
+        {
+            find: "getSrc(){",
+            replacement: {
+                match: /(if\(null!=.{1,2}\))return .{1,2}\.default\.getURL\(.{1,2}\)/,
+                replace: "$1return null",
+            },
+        },
+        {
+            find: ",findInlineEmojisFromSurrogates:",
+            replacement: {
+                match: /if\(!0!==.{1,2}&&!.{1,2}\.test\(.{1,2}\)\)/,
+                replace: "",
```

Comment:
```suggestion
                replace: "if (true)",
```
feels slightly better? /shrug

Reply:
I like it the way it is, I can change it if it's preffered though. (There also might be a minimal performance difference, but that'd be a microoptimization so it's not like it really matters).

Reply:
It's not about performance, IMO it's clearer - and it also won't break if an else is added.
I'm not a maintainer so it's up to you - or a maintainer once they see this.

Reply:
The performance part was just a sidenote (as I said, I know it doesn't really matter), it's just that in my opinion removing the `if` entirely is cleaner... It's ultimately a matter of opinion so I'm gonna leave it up to the maintainers for now. Though  your point that it will break when an `else` is added is fair...